app/assets/stylesheets/home.sass

```diff
@@ -140,41 +247,33 @@ p
   a
     color: $white
 
-.home-content
-  max-width: 1300px
-  margin: 0 auto
-  padding: 40px 30px 90px
-
 .mobile, i
   display: none
 
-.split-home
+.split
   display: flex
   align-items: center
   justify-content: space-between
   flex-wrap: wrap
-
-  // TODO: standardize the below amongst split-home and content
-  max-width: 1200px
   margin: 0 auto
+  transition: 0.1s linear margin
 
-  @media (max-width: $breakpoint-tablet)
-    .split-left
-      margin: 30px auto 0
-      width: 100%
-      max-width: 500px
-    .split-right
-      width: 100%
-      text-align: center
-      margin-bottom: 30px
+  #home-left
+    margin-left: 55px
+    margin-top: 90px
 
-  @media (min-width: $breakpoint-tablet)
-    .split-left
-      flex: 1 1 450px
-      padding-right: 10%
-    .split-right
-      flex: 0 0 auto
-      margin-right: 30px
+  .split-left
+    flex: 1 1 390px
+    padding-right: 10px
+    padding-left: 15px
+    display: inline-block
```

Comment:
`display: inline-block` shouldn't be necessary - positioning & sizing is already handled by flexbox